src/modules/displayScore.js

```diff
@@ -4,9 +4,9 @@ const scoreContainer = document.querySelector('.scores-container');
 
 const printScore = async () => {
   const result = await getScore();
```

Comment:
- [ ] When you click on run this method printScore() more than one time then it print duplicate scores 😨 as mentioned by the previous reviewer. You can fix this issue by add this code after line 5 which is shown in the image below ⬇️
![line](https://user-images.githubusercontent.com/93039359/210792861-d33479e7-d831-40dc-b3ea-840da4f6dbf1.PNG)